### ISSUE TITLE
Changing offline MOTD

### ISSUE THREAD

**Issue by farwl** [Minehut/Meta#31]
### Feature request

**Platform**
 Website / In-game

**Describe the feature**

The default Offline server MOTD is **server is offline. start via the lobby with /join <server>**. I think it would be cool if we could change this.

**How the feature is useful**

This is a cool feature because it could be exclusive to premium users.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by drtshock** [comment]
So you want it exclusive to premium servers?

Definitely possible.

**Comment by farwl** [comment]
Yeah I think it should be a premium feature since it's not necessary/vital for all servers but it's a cool extra feature that allows a bit more customization.

**Comment by Tringlle** [comment]
Yes please
